posts/2021-12-15-grails-five-one.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Grails 5.1 Released
+date: Dec 15, 2021
+description: 
+author: Puneet Behl
+image: grails-blog-index-2.png
+---
+
+# [%title]
+
+[%author]
+
+[%date]
+
+The [Grails Foundation™](https://grails.org/foundation/index.html) is pleased to announce a new minor release: Grails framework 5.1!
+
+This release of the Grails framework includes a number of bug fixes and improvements for Gradle 7.2 support, plus some updates to our plugins and dependencies.
+
+The dependencies that have been updated include:
+
+- Spring Boot 2.6.1
+- Micronaut 3.2
+- GORM for Hibernate 7.2, which updates to Hibernate 5.6
+- GORM for MongoDB 7.2, which updates to underline Mongo driver 4.4
+
+Check-out the release notes from these libraries to learn about the new features that are included with those releases.
+
+A major change that accompanies Spring Boot 2.6 is that circular references are prohibited. In order to maintain backwards compatibility for Grails users, Grails framework 5.1 explicitly enables circular references by default. This will no longer be the case once Grails framework 6 is released.  
```

Comment:
Can you link to a documentation or blog post in Spring Boot explaining that change. 

what's the setting for this in Grails?. Can users already disable them in Grails 5 applications to see if they rea ready for Grails 6.